scripts/ticketmaster-API.py

```diff
@@ -81,14 +86,19 @@
                   'name'] if (artists and 'classifications' in artists[
                   0] and 'genre' in artists[0]['classifications'][0]) else ''
 
-                query = """INSERT INTO ticketmaster_events
-                            (ticketmaster_id, local_date, event_genre,
-                            event_subgenre, venue, venue_lat, venue_long,
-                            artist_id, artist_name, artist_genre) VALUES
-                            (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)"""
-                values = (event_id, local_date, main_genre_name, sub_genre_name,
-                          main_venue_name, main_venue_lat, main_venue_lon,
-                          main_artist_id, main_artist_name, main_artist_genre)
-
-                cursor.execute(query, values)
-                db.commit()
+                entry = entry + 1
+
+                if entry == 100:
```

Comment:
[RC]
(Remove?) I feel like all of this is debugging?

Reply:
Yeah, removed